fetch_pyqt_windows.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
I can certainly move this file.

The .cfg file can live in a subdirectory, but in that case it would package the *installed* cola module, not the one from the source tree - Pynsist looks in the directory where the .cfg file is for packages. We could have a bit of extra control, like a Makefile that copies `cola/` into `contrib/win32/` and then invokes pynsist. But I'm a bit wary of a situation that would make it easy to run Pynsist and successfully build an installer that contains a different version of the code than you expected. What do you think?